Magento2/Sniffs/Attributes/NewAttributesSniff.php

```diff
@@ -0,0 +1,235 @@
+<?php
+// phpcs:ignoreFile
```

Comment:
@svera thanks for the PR

It would be better to have a separate folder for PHPCompatiblity sniffs. i.e. Magento2/PHPCompatibility.

This way, the files can be excluded from the configuration file instead of using phpcs:ignoreFile annotation for every file.

Can the unit tests for these sniffs be also included?

Reply:
@sivaschenko finally I put `PHPCompatibility` as a sniffs group, CS Fixer is very strict with the sniffs structure
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial